account_move_exception/views/account_view.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record id="action_account_test_tree" model="ir.actions.act_window">
+        <field name="name">Account Exception Rules</field>
+        <field name="res_model">exception.rule</field>
+        <field name="view_mode">tree,form</field>
+        <field name="view_id" ref="base_exception.view_exception_rule_tree" />
+        <field
+            name="domain"
+        >[('model', 'in', ['account.move', 'account.move.line'])]</field>
+        <field
+            name="context"
+        >{'active_test': False, 'default_model' : 'account.move'}</field>
+    </record>
+    <menuitem
+        id="menu_account_exception_parent"
+        parent="account.product_product_menu_sellable"
+        sequence="20"
+        name="Exceptions"
+    />
+    <menuitem
+        id="menu_account_exception_parent2"
+        parent="account.product_product_menu_purchasable"
+        sequence="20"
+        name="Exceptions"
+    />
+    <menuitem
+        action="action_account_test_tree"
+        id="menu_account_test"
+        parent="menu_account_exception_parent"
+    />
+    <record id="view_move_form" model="ir.ui.view">
+        <field name="name">account_move_exception.view_move_form</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <header position="after">
+                <div
+                    groups="account.group_account_user"
+                    class="alert alert-danger alert-dismissable"
+                    role="alert"
+                    style="margin-bottom:0px;"
+                    attrs="{'invisible':[('main_exception_id','=', False)]}"
+                >
+                    <bold>You have an outstanding
+                    exception to manage:
+                    <field
+                            name="main_exception_id"
+                            options='{"no_open": True}'
+                        /></bold>
+                </div>
+            </header>
+            <xpath expr="//field[@name='date']/.." position="inside">
+                <field
+                    name="ignore_exception"
+                    states="account"
+                    groups='base_exception.group_exception_rule_manager'
+                />
+            </xpath>
+            <xpath expr="//field[@name='company_id']" position="after">
+                <field name="exception_ids" widget="many2many_tags" readonly="True" />
```

Comment:
Are exceptions dependent on the post_block_id field? Maybe the field could be hidden if no blocking is done.

Reply:
no

Reply:
@JordiMForgeFlow  could you check now if this solves your visibility issues?